canonical-websites/www.ubuntu.com
# 2666: Move /cloud/public-cloud to /public-cloud and update copy

## Done

- Moved /cloud/public-cloud to /public-cloud
- Updated copy according to [copydoc](https://docs.google.com/document/d/1Bttn4V27eEJjgWzMSY4w9nISTQvvaNYq9CvoWXPOYPs/edit#)

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8001/public-cloud
- Check that it matches the [copydoc](https://docs.google.com/document/d/1Bttn4V27eEJjgWzMSY4w9nISTQvvaNYq9CvoWXPOYPs/edit#)
- Check that /cloud/public-cloud redirects to /public-cloud

## Issue / Card

Fixes #2666
